cli/src/movement_util.rs

```diff
@@ -110,8 +127,27 @@ fn get_target_commit(
     has_conflict: bool,
     change_offset: u64,
 ) -> Result<Commit, CommandError> {
+    let wc_revset = RevsetExpression::commit(working_commit_id.clone());
+    // If we're editing, start at the working-copy commit. Otherwise, start from
+    // its direct parent(s).
+    let start_revset = if edit {
+        wc_revset.clone()
+    } else {
+        wc_revset.parents()
+    };
+
+    let start_commit: Commit = start_revset
+        .clone()
+        .evaluate_programmatic(workspace_command.repo().as_ref())?
+        .iter()
+        .commits(workspace_command.repo().store())
+        .collect::<Result<Vec<_>, BackendError>>()?
+        .pop()
```

Comment:
Better to do `.first()` than to collect into a vector and take only the first argument.

It's also a bit weird that we ignore any other parents than the first. Makes me wonder if we should just say "from the parents" instead of the specific change id.